h2o-core/src/main/java/water/util/FileUtils.java

```diff
@@ -32,7 +30,7 @@ public static void copyStream(InputStream is, OutputStream os, final int buffer_
         os.write(bytes, 0, count);
       }
     }
-    catch(EofException eofe) {
```

Comment:
The latest Jetty exception is still the same: https://github.com/eclipse/jetty.project/blob/0c8273f2ca1f9bf2064cd9c4c939d2546443f759/jetty-io/src/main/java/org/eclipse/jetty/io/EofException.java